railgun/test/performance/perf_knative_test.go

```diff
@@ -0,0 +1,258 @@
+//+build performance_tests
+
+package performance
+
+import (
+	"crypto/tls"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	"bytes"
+	"context"
+	"fmt"
+	"testing"
+
+	types "k8s.io/apimachinery/pkg/types"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/kind"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	knativenetworkingversioned "knative.dev/networking/pkg/client/clientset/versioned"
+	"knative.dev/pkg/apis"
+	knservingv1 "knative.dev/serving/pkg/apis/serving/v1"
+	knativeversioned "knative.dev/serving/pkg/client/clientset/versioned"
```

Comment:
`goimports -local github.com/kong`
```suggestion
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
	types "k8s.io/apimachinery/pkg/types"
	corev1 "k8s.io/api/core/v1"
	v1 "k8s.io/api/core/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	knativenetworkingversioned "knative.dev/networking/pkg/client/clientset/versioned"
	"knative.dev/pkg/apis"
	knservingv1 "knative.dev/serving/pkg/apis/serving/v1"
	knativeversioned "knative.dev/serving/pkg/client/clientset/versioned"
	
	"github.com/kong/kubernetes-testing-framework/pkg/kind"
```